changes/30.3.3.md

```diff
@@ -0,0 +1 @@
+* Make Reversed S (`U+01A7`..`U+01A8`, `U+A644`..`U+A645`) and Italic Cyrillic Lower Ghe (`U+0433`, `U+0453`, `U+0493`, `U+04F7`, `U+04FB`) follow variants of Cyrillic Ze (`cv69`..`cv70`).
```

Comment:
You should make a new selector rather than make it follow Cyrillic Ze, as reverse S is very different from Ze shapes.